src/server/wesnothd/server.cpp

```diff
@@ -3094,5 +3094,12 @@ int main(int argc, char** argv)
 		}
 	}
 
+const std::string title = "wesnoth server";
+#ifdef WIN32
+	SetConsoleTitleA(title.c_str());
+#else
+	std::cout << "\033]0;" << title << "\007";
```

Comment:
I think this was suggested because it looked like a window title was set on windows before but the whole cmd call was only there to not actually show a window so...

I don't mind if we want to do this either way but then we should check `isatty` before writing this out.

Reply:
Honestly I'd prefer to just not set the title - if someone is starting wesnothd from the command line then they'd already know what it's open for.

Reply:
I personally think that this is nice to have, though not essential. I definitely don't mind calling `isatty` before writing terminal escape codes.

Certainly they'll know what it's open for if they launch it from the command-line… but that's assuming they launch it manually (for example, not some automatic script run on launch) and also that they launched it recently (for example, they didn't leave for a 3 month vacation and come back). Neither of those two examples are cases that absolutely need this, but it's easy to do and, in my opinion, just nice to have.

Reply:
Regardless of whether we want to do this, I wouldn't say it necessarily has to be here in this PR.